src/core/configuration.ts

```diff
@@ -208,7 +210,12 @@ function bindOptionWatchers(config: Configuration, vueInstance: Pick<Vue, "$watc
     const targets = config.getOptionsToWatch();
     if (targets) {
         targets.forEach((optionName: string) => {
-            vueInstance.$watch(optionName, (value) => config.updateValue(optionName, value));
+            vueInstance.$watch(optionName, (value) => {
+                if (config.changedOptions[optionName] !== value) {
```

Comment:
Better make the `changedOptions` an argument shared between `bindOptionWatchers` and `subscribeOnUpdates`